tests/lib/pkgdb.sh

```diff
@@ -490,6 +552,30 @@ pkg_dependencies_opensuse(){
         "
 }
 
+pkg_dependencies_arch(){
+    echo "
+    curl
```

Comment:
Curl? :)

Reply:
Yeah, I can drop it, it's installed in spread.yaml anyway.

Reply:
OTOH, we seem to install it for other distros here anyway.